SendLog.py

```diff
@@ -75,10 +94,9 @@ def parse_line(line):
 		return (figure_id, ENDING)
 	else:
 		# Current figure coordinates
-		figure_id = int(line_words[2])
+		figure_id = int(line_words[3])
 		return (figure_id, DATA)
-		
-def get_line_and_state(file):
+def get_line_and_state_modified(file):
 	# Function to get a new line from file
 	# This also parses the line and appends new figures to figure List
 	global figure_list
```

Comment:
Avoid global variables at all cost. Please figure out if there is a better way to do this. Can you use a Database?